core/arch/arm/tee/entry_std.c

```diff
@@ -292,6 +325,9 @@ static void entry_open_session(struct thread_smc_args *smc_args,
 	plat_prng_add_jitter_entropy();
 
 out:
+	cleanup_params(arg->params + num_meta, saved_attr,
```

Comment:
Can you cross check that there is no issue when this cleanup from `get_open_session_meta()`. `num_meta` may not be inited ?